Develop (8.x) worker profile does not boot

**Describe the bug**
A build of develop's worker profile (tested against 7c4a9496276bce2fd3d5f6dbfa5a8e058647b28f) spits the following on startup:
```
2019-09-24T20:09:05,444 | ERROR | (BootFeaturesInstaller:113) - Error installing boot features
org.apache.felix.resolver.reason.ReasonException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=opencast-worker; type=karaf.feature; version="[8.0.0.SNAPSHOT,8.0.0.SNAPSHOT]"; filter:="(&(osgi.identity=opencast-worker)(type=karaf.feature)(version>=8.0.0.SNAPSHOT)(version<=8.0.0.SNAPSHOT))" [caused by: Unable to resolve opencast-worker/8.0.0.SNAPSHOT: missing requirement [opencast-worker/8.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-services-processing-light-load; type=karaf.feature [caused by: Unable to resolve opencast-services-processing-light-load/8.0.0.SNAPSHOT: missing requirement [opencast-services-processing-light-load/8.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-videoeditor-workflowoperation; type=osgi.bundle; version="[8.0.0.SNAPSHOT,8.0.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT: missing requirement [opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=org.opencastproject.workflow.handler.workflow)(version>=8.0.0)(!(version>=9.0.0)))"]]]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:392) ~[?:?]
        at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:378) ~[?:?]
        at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:332) ~[?:?]
        at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:257) ~[?:?]
        at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:388) ~[?:?]
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1025) ~[?:?]
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:964) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: org.apache.felix.resolver.reason.ReasonException: Unable to resolve opencast-worker/8.0.0.SNAPSHOT: missing requirement [opencast-worker/8.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-services-processing-light-load; type=karaf.feature [caused by: Unable to resolve opencast-services-processing-light-load/8.0.0.SNAPSHOT: missing requirement [opencast-services-processing-light-load/8.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-videoeditor-workflowoperation; type=osgi.bundle; version="[8.0.0.SNAPSHOT,8.0.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT: missing requirement [opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=org.opencastproject.workflow.handler.workflow)(version>=8.0.0)(!(version>=9.0.0)))"]]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        ... 12 more
Caused by: org.apache.felix.resolver.reason.ReasonException: Unable to resolve opencast-services-processing-light-load/8.0.0.SNAPSHOT: missing requirement [opencast-services-processing-light-load/8.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-videoeditor-workflowoperation; type=osgi.bundle; version="[8.0.0.SNAPSHOT,8.0.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT: missing requirement [opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=org.opencastproject.workflow.handler.workflow)(version>=8.0.0)(!(version>=9.0.0)))"]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        ... 12 more
Caused by: org.apache.felix.resolver.reason.ReasonException: Unable to resolve opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT: missing requirement [opencast-videoeditor-workflowoperation/8.0.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=org.opencastproject.workflow.handler.workflow)(version>=8.0.0)(!(version>=9.0.0)))"
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343) ~[?:?]
        ... 12 more

```

**To Reproduce**
Steps to reproduce the behavior:
1. Build develop
2. Attempt to start the worker

**Environment Information**
 - OS: Debian 9, Debian 10
 - Opencast version with minor release: Develop (currently 8.x)

--- a/assemblies/karaf-features/src/main/feature/feature.xml
+++ b/assemblies/karaf-features/src/main/feature/feature.xml
@@ -573,17 +573,14 @@
     <bundle start-level="82">mvn:org.opencastproject/opencast-textextractor-tesseract/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-timelinepreviews-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-timelinepreviews-ffmpeg/${project.version}</bundle>
-    <bundle start-level="82">mvn:org.opencastproject/opencast-timelinepreviews-workflowoperation/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-transcription-service-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-transcription-service-google-speech-impl/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-transcription-service-ibm-watson-impl/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-transcription-service-persistence/${project.version}</bundle>
-    <bundle start-level="82">mvn:org.opencastproject/opencast-transcription-service-workflowoperation/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-usertracking-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-usertracking-impl/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-videoeditor-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-videoeditor-ffmpeg-impl/${project.version}</bundle>
-    <bundle start-level="82">mvn:org.opencastproject/opencast-videoeditor-workflowoperation/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-videosegmenter-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-videosegmenter-ffmpeg/${project.version}</bundle>
   </feature>

